desc/geometry/core.py

```diff
@@ -142,20 +167,20 @@ def compute(
         )
         return data
 
-    def translate(self, displacement=[0, 0, 0]):
-        """Translate the curve by a rigid displacement in x, y, z."""
-        self.shift += jnp.asarray(displacement)
+    def translate(self, displacement):
+        """Translate the curve by a rigid displacement in X, Y, Z."""
+        self.shift = self.shift + np.asarray(displacement)
 
-    def rotate(self, axis=[0, 0, 1], angle=0):
-        """Rotate the curve by a fixed angle about axis in xyz coordinates."""
-        R = rotation_matrix(axis, angle)
-        self.rotmat = R @ self.rotmat
+    def rotate(self, axis):
+        """Rotate the curve about axis in X, Y, Z coordinates."""
```

Comment:
should mention that the angle is determined by the norm of axis

Reply:
I am slightly confused, so now if I want to rotate about the vector (1,1,0) 90 degrees (pi/4), I should pass in for the axis (1,1,0) * np.sqrt(pi/4 / 2)?